content/show/this-week-in-bitcoin/_index.md

```diff
@@ -14,10 +14,10 @@ header_image = "/images/shows/this-week-in-bitcoin.png"
 
 podverse_podcast_id = "MxEmZU_zDt"
 
-[links.shownotes]
-  url="https://www.thisweekinbitcoin.show"
-[links.email]
-  url="/contact"
+[links.episodesfm]
+  url="https://episodes.fm/1733733790"
```

Comment:
I wonder if this should be more of the individual show site link since it uses the `podcast` icon. I think that would look better than the `shownotes` link that we use currently.

```suggestion
[links.show-site]
  url="https://episodes.fm/1733733790"
```

Reply:
I just feel like episode.fm might be too specific (much like how we have fireside_url in episode pages even though most of our shows don't come from there anymore). I just want it to stay generic enough to where we don't need to update it to something else down the road if we don't have to.

Reply:
> I just want it to stay generic enough to where we don't need to update it to something else down the road if we don't have to.

Totally on board with this. I wonder about names... Thinking about what episodesfm is doing maybe some ideas:
- listen link
- podcatcher
- app link
- directory
- universal link
- smart link

But I'm fine with whatever.

Reply:
@CGBassPlayer thoughts on any of these? Still prefer `show-site`? I'm down for whatever just want to poke this along.

Reply:
I like the `universal-link` or the `smart-link`

I think either of those would work well and that would be good if you guys move away from episodes.fm for whatever reason

Reply:
Just updated my suggestions to `smart-link`

Reply:
Updated to smart-link.